miner/miner.go

```diff
@@ -288,16 +288,10 @@ minerLoop:
 			//
 			// See:  https://github.com/filecoin-project/lotus/issues/1845
 			nextRound := time.Unix(int64(base.TipSet.MinTimestamp()+build.BlockDelaySecs*uint64(base.NullRounds))+int64(build.PropagationDelaySecs), 0)
-
-			select {
-			case <-build.Clock.After(build.Clock.Until(nextRound)):
```

Comment:
I don't think this is right, we explicitly need to handle the stop case here

Reply:
We also handle stopping at the top of this loop

Reply:
It might be better to use functions